sirepo/job_api.py

```diff
@@ -275,7 +275,9 @@ def _content_and_uri(kwargs):
                     f"expected json content-type={r.headers['content-type']}"
                 )
             j = pkjson.load_any(r.body)
-            if d and sirepo.job.is_ok_reply(j):
+            if d and (
+                sirepo.job.is_ok_reply(j) or j.get("state") == sirepo.job.COMPLETED
```

Comment:
I think "COMPLETED" could be considered an is_ok_reply.  It's only used in this one place.

Reply:
30d034f27